### ISSUE TITLE
Update components to new focus rules

### ISSUE THREAD

**Issue by mimarz** [equinor/design-system#284]
After review from designer it has been decided that all focus border should be the same, as in a square. Also on click should not trigger focus, only keyboard navigation.

Reference commentary here: https://www.figma.com/file/BleNV0tE9Aopz3asajjNkA/Bendik-test?node-id=1564%3A0

All focus outlines are to be `dashed` `1px width, 2px offsett`

Apply `z-index: 1` where needed for overlapping components.

*Following components needs to be fixed with new "focus rules"*

- [x] Chips
- [x] Ghost Icon button
- [x] Button: Don't show focus on click
- [x] Tabs: Update focus offsett
